netlify/functions/serverless/index.js

```diff
@@ -6,7 +6,8 @@ const cleanseQueryParameters = require('./cleanse-query-parameters');
 require('./eleventy-bundler-modules.js');
 
 /**
- * @param event
+ * @param event - 11ty serverless event
+ * @returns object
```

Comment:
If we want to be specific really specific about `event`'s type, we could:

1. `npm install --save-dev @netlify/functions`
2. Replace this bit with:

```suggestion
 * @type {import('@netlify/functions').Handler}
 * @param event - serverless function call event
 * @returns HTTP response with a page body
```